api/src/main/java/io/strimzi/api/kafka/model/AbstractKafkaConnectSpec.java

```diff
@@ -78,7 +78,9 @@ public void setVersion(String version) {
         this.version = version;
     }
 
-    @Description("The docker image for the pods.")
+    @Description("The Docker image used for generating Kafka Connect pods. "
+        + "If no image name is explicitly specified, it is determined based on the `version` configuration. "
+        + "If a version is not configured, the image name corresponds to the default version specified in the Cluster Operator configuration.")
```

Comment:
The image corresponds to the `Cluster Operator configuration` even when the `.spec.version` field is specified.  I would either leave the whole version aspect out or clarify this.

Reply:
I've changed instances of this sentence to read: _The image names are specifically mapped to corresponding versions in the Cluster Operator configuration._